### PR TITLE
Updated NuGet packages

### PR DESCRIPTION
**Pull request by jamesmcroft**
## Resolves #

Updated NuGet packages

## PR checklist

- [ ] Have samples been added or updated, run locally, and work as intended
- [ ] Have code styling rules been run on all new source file changes
- [ ] Have relevant articles in the docs been added or updated for all new source file changes
- [ ] Have major breaking changes been made and are documented

### CODE REVIEW

File: src/MADE.Media.Image/MADE.Media.Image.csproj [hunk 1]
```diff
@@ -29,4 +29,8 @@
     </Page>
   </ItemGroup>
 
+  <ItemGroup>
```
Comment (tom-made) [inline src/MADE.Media.Image/MADE.Media.Image.csproj:32]: Unsure if this was meant to have been added here

File: src/MADE.UI.Controls.ChipBox/MADE.UI.Controls.ChipBox.csproj [hunk 2]
```diff
@@ -26,6 +26,10 @@
     </Page>
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Uno.UI" Version="4.4.13" />
```
Comment (tom-made) [inline src/MADE.UI.Controls.ChipBox/MADE.UI.Controls.ChipBox.csproj:30]: Similarly here.

**Review by tom-made — CHANGES_REQUESTED** [review]
Worth taking a look to make sure the package update worked. Appears there are some unnecessary changes

**Comment by jamesmcroft** [comment]
@tom-made I see what's happened. VS package manager didn't update the conditional package 🤦

**Review by tom-made — APPROVED** [review]

**Pull request merged**